Source/Fuse.Nodes/Node.uno

```diff
@@ -133,5 +133,28 @@ namespace Fuse
 			return this;
 		}
 
+		//We want `: Behavior` here. This is an attempt to workaround a Uno/DotNet issue on some platforms
```

Comment:
I'd just file a ticket with out a full set of information (or maybe to a branch of fuselibs wich reproduces it).